src/screens/LockedWallet.js

```diff
@@ -120,7 +136,13 @@ class LockedWallet extends React.Component {
             {this.state.errorMessage && <p className="mt-4 text-danger">{this.state.errorMessage}</p>}
             <div className="d-flex align-items-center justify-content-between flex-row w-100 mt-4">
               <a className="mt-4" onClick={(e) => this.resetClicked(e)} href="true">{t`Reset all data`}</a>
-              <button onClick={this.unlockClicked} type="button" className="btn btn-hathor">{t`Unlock`}</button>
+              <button onClick={this.unlockClicked} type="button" className="btn btn-hathor">
+                {
+                  this.state.loading ?
+                  <ReactLoading type='spin' width={24} height={24} /> :
```

Comment:
Can you add a video how is the UI with the loading inside the button please?

Reply:
Added the video to the PR description